helm-charts/falcon-sensor/templates/container_deployment_webhook.yaml

```diff
@@ -84,6 +84,17 @@ spec:
     {{- end }}
     spec:
       affinity:
+        podAntiAffinity:
```

Comment:
How would you feel about making these affinity statements default values, and allowing for them to be overridden?